README.md

```diff
@@ -34,7 +34,7 @@ It came from a desire to quickly dive into graphics and performance experiments.
 
 ## Requirements
 * Windows 10
-* [Visual Studio 2017](https://www.visualstudio.com/) with the [Windows 10 April 2018 Update SDK](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
+* [Visual Studio 2017](https://www.visualstudio.com/) with the [Windows 10 October 2018 Update (version 1809) SDK](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
```

Comment:
17763 instead of 1809 for SDKs is more clear in my opinion as that's whats also how it shows up during VS install. That's also how DXR samples reference it.